docs/topics/multiplatform/multiplatform-publish-lib.md

```diff
@@ -4,6 +4,10 @@ You can publish a multiplatform library to a local Maven repository with the [`m
 Specify the group, version, and the [repositories](https://docs.gradle.org/current/userguide/publishing_maven.html#publishing_maven:repositories) 
 where the library should be published. The plugin creates publications automatically.
 
+> Here and further down the page all examples show the `shared/build.gradle.kts` file, not the root project configuration.
```

Comment:
I think it would be better to repeat this in every section with a code sample  (note as a note, but as an action), for example:
> In the `shared/build.gradle.kts` file, specify the group, version..